das_client/lib/logging/src/das_log_tree.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:io';
+
+import 'package:das_client/logging/src/log_entry.dart';
+import 'package:das_client/logging/src/log_level.dart';
+import 'package:das_client/logging/src/log_service.dart';
+import 'package:das_client/util/device_id_info.dart';
+import 'package:device_info_plus/device_info_plus.dart';
+import 'package:fimber/fimber.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class DasLogTree extends LogTree {
+  final LogService _logService;
+  late Future<void> _initialized;
+  final Map<String, String> metadata = {};
+
+  DasLogTree({required LogService logService}) : _logService = logService {
+    _initialized = _init();
+  }
+
+  Future<void> _init() async {
+    Fimber.i('Initializing DasLogTree...');
+    metadata['deviceId'] = await DeviceIdInfo.getDeviceId();
+
+    final info = await PackageInfo.fromPlatform();
+    metadata['appVersion'] = info.version;
+
+    final deviceInfoPlugin = DeviceInfoPlugin();
+    if (Platform.isAndroid) {
+      _processAndroidDeviceInfo(await deviceInfoPlugin.androidInfo);
+    } else if (Platform.isIOS) {
+      _processIosDeviceInfo(await deviceInfoPlugin.iosInfo);
+    }
+  }
+
+  void _processAndroidDeviceInfo(AndroidDeviceInfo deviceInfo) {
+    metadata['systemName'] = "android";
```

Comment:
I think it would make sense to use either single or double quotes in the code or do you have a specific system? Could be linted with [prefer_single_quotes](https://dart.dev/tools/linter-rules/prefer_single_quotes) or [prefer_double_quotes](https://dart.dev/tools/linter-rules/prefer_double_quotes). Personally, I prefer single quotes.